lib/ffmpeg/vaapi/encoder.py

```diff
@@ -92,7 +98,7 @@ def check_output(self):
     ipbmode = 0 if vars(self).get("gop", 0) <= 1 else 1 if vars(self).get("bframes", 0) < 1 else 2
     ipbmsgs = [
       "Using intra frames only",
-      "Using intra and P-frames|[L|l]ow delay|forward-prediction"
+      "|Using intra and P-frames|[L|l]ow delay|forward-prediction"
```

Comment:
Why?  This is a new entry in the list... why put an `|` operator at the start?

Reply:
Removed. This is not needed